src/components/Banner/Banner.styles.js

```diff
@@ -50,6 +50,15 @@ export const Container = styled.div`
     background-color: ${getThemeValue("primary", "lightest")};
     border-color: ${getThemeValue("primary", "base")};
   }
+
+  &.banner-variant-custom {
+    background-color: ${props => props.customColors.backgroundColor};
+    border-color: ${props => props.customColors.primaryColor};
+
+    .text--primary {
+      color: ${props => props.customColors.textColor} !important;
+    }
```

Comment:
Let's apply this on the styled-component that holds text , so that's it's more explicit, and we don't override with !important

https://github.com/ticketmaster/aurora/blob/fe52c244960bd12ba095a7bd0f0dbf484e3d3d20/src/components/Banner/Banner.styles.js#L69
https://github.com/ticketmaster/aurora/blob/fe52c244960bd12ba095a7bd0f0dbf484e3d3d20/src/components/Banner/index.js#L198